release-notes-3.0.0.md

```diff
@@ -0,0 +1,13 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### Breaking Changes
+* [SCMOD-5747](https://autjira.microfocus.com/browse/SCMOD-5747) - The workflow worker does not use the processing-service and the
+workflow-admin anymore. Workflows are loaded from files saved on disk (they are in the [darwin-workflow-worker](https://github.houston.softwaregrp.net/Verity/darwin-workflow-worker)).
```

Comment:
Don't point to the closed source worker for this. Can you point to where the files need to be in this worker or better still point to the documentation which describes where these files need to be and what they are?